tests/optional_kernel_features/kernel_features_mismatched_nd_range_exception.cpp

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for the exception that is thrown by
+//  [[sycl::reqd_work_group_size]] attribute when use nd_range of wrong size in
+//  kernel.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "catch2/catch_template_test_macros.hpp"
+
+namespace kernel_features_mismatched_nd_range_exception {
+using namespace sycl_cts;
+
+struct kernel_separate_lambda;
+struct kernel_functor;
+struct kernel_submission_call;
+
+// Define required size of work group for attribute
+constexpr int testing_wg_size = 16;
+
+class Functor {
+ public:
+  [[sycl::reqd_work_group_size(testing_wg_size)]] void operator()(
+      sycl::nd_item<1>) const {}
+};
+
+TEST_CASE(
+    "Kernel features. Exceptions throwns by [[reqd_work_group_size()]] with "
+    "mismatched nd_range",
```

Comment:
Suggest rewording to something like `Submitting a kernel with an nd-range that does not match [[reqd_work_group_size()]] throws an exception`.

Reply:
Because tag doesn't appear in output, when test fails. By adding this prefix I wanted to make test output more clear for enduser.

Reply:
I see. IMO it's still a bit redundant because the file name is listed directly below the failing test case description:
![image](https://user-images.githubusercontent.com/791348/155498683-1d0cc1d9-6af7-4ed4-9687-287adff157d5.png)

If clarity is a major concern we could also investigate adding a custom console reporter that groups test cases more visibly into their respective categories.

Reply:
Okay. I thought we talked about tag `[kernel_features]` from the second argument. Looks like I misunderstood you. 
I agree, it looks redundant. I will remove the prefix from the test name.